src/nvim/mark.h

```diff
@@ -5,6 +5,34 @@
 #include "nvim/mark_defs.h"
 #include "nvim/pos.h"
 
+/// Return true if position a is before (less than) position b.
+inline bool lt(pos_T a, pos_T b) FUNC_ATTR_CONST {
+    if (a.lnum != b.lnum) {
+        return a.lnum < b.lnum;
+    } else if (a.col != b.col) {
```

Comment:
This doesn't follow our style guide, please see http://neovim.io/develop/style-guide.xml

It should be a fairly quick read, although the only things which look off in this function are:
- the indentation should use two spaces
- we use the [Linux kernel variant of K&R](https://en.wikipedia.org/wiki/Indent_style#Variant:_Linux_kernel) for indenting